sparkle/src/components/IconButton.tsx

```diff
@@ -42,6 +42,11 @@ const styleVariants: Record<ButtonVariantType, string> = {
     "hover:s-text-slate-100 hover:s-text-slate-100" +
     "active:s-text-slate-200 active:s-text-slate-200" +
     "s-text-white/50 s-text-white/50",
+  "ghost-secondary":
+    "s-text-white s-text-white" +
+    "hover:s-text-slate-100 hover:s-text-slate-100" +
+    "active:s-text-slate-200 active:s-text-slate-200" +
+    "s-text-white/50 s-text-white/50",
```

Comment:
Okay I think we have a bunch of duplicates here 😅 Would you mind getting rid of it?

Reply:
I'd love to.
I don't know why IconButton's variants are mapped to Button ones, it makes no sense.

I don't remember doing it so I did not revert.

Reply:
Since we want to retire them, I figured it did not require addressing.